liquidctl/driver/asus_gpu.py

```diff
@@ -0,0 +1,184 @@
+"""liquidctl drivers for ASUS ROG NVIDIA graphics cards.
+
+Copyright (C) 2020–2020  Marshall Asch and contributors
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+from enum import Enum, unique
+import logging
+import re
+
+from liquidctl.driver.smbus import SmbusDriver
+from liquidctl.error import NotSupportedByDevice
+
+_LOGGER = logging.getLogger(__name__)
+
+_NVIDIA = 0x10de                # vendor
+_ASUS = 0x1043                  # subsystem vendor
+
+_ROG_RTX_2080_TI = 0x1e07       # device id
```

Comment:
I moved the subsystem device id into the `probe()` function

Reply:
Sorry, I just realized it's necessary to have it in the scope of the module, because the test cases need to access them (and I want to limit the amount of private things the tests rely on).